src/main/kotlin/app/revanced/patches/youtube/misc/videobuffer/patch/CustomVideoBufferPatch.kt

```diff
@@ -1,201 +1,37 @@
 @Name("custom-video-buffer")
 @Description("Lets you change the buffers of videos.")
 @DependsOn([SettingsPatch::class])
 @CustomVideoBufferCompatibility
 @Version("0.0.1")
-class CustomVideoBufferPatch : BytecodePatch(
-    listOf(
-        InvokeMaxBufferFingerprint,
-        PlaybackBufferFingerprint,
-        ReBufferFingerprint,
-    )
-) {
+class CustomVideoBufferPatch : BytecodePatch() {
     override fun execute(context: BytecodeContext): PatchResult {
         SettingsPatch.PreferenceScreen.MISC.addPreferences(
-            PreferenceScreen(
-                "revanced_custom_video_buffer",
-                StringResource("revanced_custom_video_buffer_title", "Video buffer settings"),
-                listOf(
-                    TextPreference(
-                        "revanced_pref_max_buffer_ms",
-                        StringResource("revanced_pref_max_buffer_ms_title", "Maximum buffer size"),
-                        InputType.NUMBER,
-                        "120000",
-                        StringResource(
-                            "revanced_pref_max_buffer_ms_summary",
-                            "The maximum size of a buffer for playback"
-                        )
-                    ),
-                    TextPreference(
-                        "revanced_pref_buffer_for_playback_ms",
-                        StringResource("revanced_pref_buffer_for_playback_ms_title", "Maximum buffer for playback"),
-                        InputType.NUMBER,
-                        "2500",
-                        StringResource(
-                            "revanced_pref_buffer_for_playback_ms_summary",
-                            "Maximum size of a buffer for playback"
-                        )
-                    ),
-                    TextPreference(
-                        "revanced_pref_buffer_for_playback_after_rebuffer_ms",
-                        StringResource(
-                            "revanced_pref_buffer_for_playback_after_rebuffer_ms_title",
-                            "Maximum buffer for playback after rebuffer"
-                        ),
-                        InputType.NUMBER,
-                        "5000",
-                        StringResource(
-                            "revanced_pref_buffer_for_playback_after_rebuffer_ms_summary",
-                            "Maximum size of a buffer for playback after rebuffering"
-                        )
-                    )
-                ),
-                StringResource("revanced_custom_video_buffer_summary", "Custom settings for video buffer")
+            NonInteractivePreference(
+                StringResource("revanced_custom_video_buffer_disclaimer_title", "Custom video buffer"),
+                StringResource("revanced_custom_video_buffer_disclaimer_title_summary",
+                    "Notice: Due to recent changes by YouTube, custom video buffer no longer functions"
+                            + " and the patch was removed"),
```

Comment:
How about this?
```suggestion
                    "These settings have been removed, because they were not functional for the duration of their existence"),
```